backend/fac-backup-util.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+set -e
+source tools/util_startup.sh
+version=$1
+run_option=$2
+s3_name="fac-private-s3"
+backup_s3_name="backups"
+db_name="fac-db"
+backup_db_name="fac-snapshot-db"
+initial_date=$(date +%Y%m%d%H%M)
+scheduled_date=$(date +%m-%d-%H)
+daily_date=$(date +%m-%d)
+mkdir tmp && cd tmp || return
+
+GetUtil() {
+    curl -x "$https_proxy" -L "https://github.com/GSA-TTS/fac-backup-utility/releases/download/$version/gov.gsa.fac.cgov-util-$version-linux-amd64.tar.gz" -O
+    tar -xvf "gov.gsa.fac.cgov-util-$version-linux-amd64.tar.gz" && rm "gov.gsa.fac.cgov-util-$version-linux-amd64.tar.gz"
+}
+InstallAWS() {
+    ./gov.gsa.fac.cgov-util install_aws
+}
+AWSS3Sync() {
+    ./gov.gsa.fac.cgov-util s3_sync --source_s3 s3://"$1"/ --dest_s3 s3://"$2"/
+}
+RDSToS3Dump() {
+    ./gov.gsa.fac.cgov-util db_to_s3 --db "$1" --s3path s3://"$2"/"$3"/
+}
+RDSToRDS() {
+    ./gov.gsa.fac.cgov-util db_to_db --src_db "$1" --dest_db "$2" --operation "$3"
+}
+
+if [ "$run_option" == "initial_backup" ]; then
+    GetUtil
+    InstallAWS
+    gonogo "install_aws"
+    RDSToS3Dump "$db_name" "$backup_s3_name" "initial/$initial_date"
+    gonogo "db_to_s3"
+    RDSToRDS "$db_name" "$backup_db_name" "initial"
```

Comment:
We need to handle the case of protected db.

`fac-db` should be considered protected, in that it doesnt do restoration to that, however, if we are going to `fac-snapshot-db` we should consider dropping table before backing up, to handle changes on the src.